app/models/georeference/verbatim_data.rb

```diff
@@ -74,6 +74,32 @@ def dwc_georeference_attributes
     h
   end
 
+  # @return [Boolean]
+  #    true if geographic_item.geo_object is completely contained in collecting_event.geographic_area
+  # .default_geographic_item
+  def check_obj_within_distance_from_area(distance)
```

Comment:
Probably nothing optimizable, but had me think a little about whether/how many times we have to hit the database here.  Would likely take more work than worthwhile at this point to tune things.